internal/service/advancedclusterold/data_source_advanced_clusters.go

```diff
@@ -1,4 +1,4 @@
-package advancedcluster
+package advancedclusterold
```

Comment:
Since we will have a few months in this state, might prefer leaving existing advanced cluster implementation with `advancedcluster` package name and have new implementation with `advancedclustertfp` so there is no confusion.

Reply:
thanks, done in this commit https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2679/commits/368c56d0baa8e849a28e2c87da7dfad210822adb